Rakefile

```diff
@@ -4,3 +4,44 @@
 require File.expand_path('../config/application', __FILE__)
 
 Osra::Application.load_tasks
+
+namespace :cucumber do
+    desc 'run all hq features'
+    desc 'run all aa features'
+    task :aa do
+      # sh 'cucumber features/aa -r features/aa -r features/support'
+      sh 'cucumber -P -v -r features/aa -r features/support features/aa'
+    end
+
+    task :hq do
+      # sh 'cucumber features/hq -r features/hq -r features/support'
+      sh 'cucumber -P -v -r features/hq -r features/support features/hq'
+    end
+end
+
+namespace :spec do
+  desc 'run all hq specs'
+    desc 'run all aa specs'
+    task :aa do
+      # sh 'rspec --exclude-pattern "spec/*/hq/**/*_spec.rb, spec/*/*/hq/**/*_spec.rb"'
+      # sh 'rspec spec/*/hq spec/*/*/hq'
+      sh 'rspec --exclude-pattern spec/\*\*\/hq{,/\*/\*\*\}/\*_spec.rb'  # model specs are also tested here here
+    end
+
+    task :hq do
+      # sh 'rspec --pattern "spec/*/hq/**/*_spec.rb, spec/*/*/hq/**/*_spec.rb"'
+      # sh 'rspec spec/*/hq spec/*/*/hq'
+      sh 'rspec --pattern spec/\*\*\/hq{,/\*/\*\*\}/\*_spec.rb'
```

Comment:
Yes, I agree. don't know why I didn't add them from the beginning.
`$rake` is running all the steps (aa and hq) in one set, so the model specs will not run twice.